roles/pulp_devel/tasks/install_podman.yml

```diff
@@ -58,7 +58,7 @@
     value: "1"
   when:
     - ansible_facts.distribution == 'Debian'
-    - ansible_virtualization_type != 'docker'
+    - ansible_virtualization_type == 'podman'
```

Comment:
This doesn't work on Ansible 2.9 (Fedora's current package):

On my Fedora 32 laptop, I ran:
```
podman run -it fedora:32 /bin/bash
dnf install -y ansible
ansible -m setup localhost | grep virtualization
```
I got:
```
        "ansible_virtualization_role": "host",
        "ansible_virtualization_type": "kvm",
```

I then ran:
```
podman run -it fedora:32 /bin/bash
dnf install -y ansible
ansible -m setup localhost | grep virtualization
```
I got:
```
        "ansible_virtualization_role": "guest",
        "ansible_virtualization_type": "docker",
```

Reply:
```suggestion
     - ansible_env.container is defined
     - ansible_env.container == 'podman' 
```

Reply:
I changed it because I was getting `kvm` which was breaking the CI.
I found these issues:
- https://github.com/ansible/ansible/issues/66304
- https://github.com/ansible/ansible/issues/64954
And decided to give a try to `podman`

Reply:
@fao89 Yeah, it looks like the podman fact is in 2.10, but not 2.9.